src/content/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-and-dt.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: New Relic Android mobile monitoring with distributed tracing
+metaDescription: This page describes how to set up Android distributed tracing.
+redirects:
+
+  - /docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-mobile-and-dt
+---
+New Relic Android mobile monitoring agents support distributed tracing. This lets you see how your mobile app activity connects to related services.
+
+Watch this short video (approx. 2:15 minutes) to learn how to:
+
+* Find mobile distributed tracing data in New Relic.
+* Filter by standard and custom attributes.
+
+<Video id="tkAxeE5cT2Q" type="youtube" />
+
+## Requirements
+
+We recommend the most current agent, but to use distributed tracing you need at least Android agent version 6.0.0.
+
+<Callout variant="tip">
+Infinite Tracing is not yet available for mobile monitoring.
+</Callout>
+
+## How to set up distributed tracing
+
+For mobile agents that support this feature, it’s enabled by default. 
+
+If you prefer to turn off distributed tracing, see the following: [Android feature flag](/docs/mobile-monitoring/new-relic-mobile-android/api-guides/android-agent-configuration-feature-flags/#dt)
```

Comment:
I'd go with: "If you prefer to turn off distributed tracing, see [Android feature flag]"

Reply:
@urbiz-nr, I used your proposed format!